mesa/space.py

```diff
@@ -215,6 +239,17 @@ def out_of_bounds(self, pos):
         x, y = pos
         return x < 0 or x >= self.width or y < 0 or y >= self.height
 
+    def iter_cell_list_contents(self, cell_list):
+        '''
+        Args:
+            cell_list: Array-like of (x, y) tuples
+
+        Returns:
+            A iterator of the contents of the cells identified in cell_list
+        '''
+        return (
```

Comment:
It returns a [generator](https://docs.python.org/3/reference/expressions.html#generator-expressions).